provision/aws/rosa_oc_login.sh

```diff
@@ -22,13 +23,6 @@ if [[ "$RUNNER_DEBUG" == "1" ]]; then
   # prevent logging the password in debug mode
   set +x
 fi
-ADMIN_PASSWORD=$(aws secretsmanager get-secret-value --region $SECRET_MANAGER_REGION --secret-id $KEYCLOAK_MASTER_PASSWORD_SECRET_NAME --query SecretString --output text --no-cli-pager)
```

Comment:
@martin-kanis I think it still makes sense to not print `asw secretmanager` commands in debug mode. Could you please go through all usages of `aws secretmanager` in our script and make sure we use 
```
if [[ "$RUNNER_DEBUG" == "1" ]]; then
  # prevent logging the password in debug mode
  set +x
fi
aws secretmanager
if [[ "$RUNNER_DEBUG" == "1" ]]; then
  set -x
fi
```
everywhere?

Reply:
Thanks, added.